src/makeCalculator.js

```diff
@@ -38,6 +38,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      calculator.result += value;
+    },
+    subtract(value) {
```

Comment:
```suggestion
    result: 0,
    
    add(value) {
      calculator.result += value;
    },
    
    subtract(value) {
```
Don't forget to provide empty lines between methods, so it is easier to read your code